bus/bus.go

```diff
@@ -1603,9 +1625,20 @@ func (b *bus) Handler() http.Handler {
 
 // Shutdown shuts down the bus.
 func (b *bus) Shutdown(ctx context.Context) error {
+	// Reset start time.
+	b.mu.Lock()
+	b.startTime = time.Time{}
```

Comment:
I don't think we need to reset the time? The API is shut down so nothing is going to consume it.
If we set it on startup and never reset it we also don't need it to be covered by the mutex.